src/components/post-test-summary-section/post-test-summary-section.ts

```diff
@@ -37,4 +39,16 @@ export class PostTestSummarySectionComponent {
     textboxModal.onDidDismiss((notes?: string) => (this.faultNotes[faultName] = notes));
     textboxModal.present();
   }
+
+  isComplete(): boolean {
+    if (!this.canComplete) {
+      return true;
+    }
+    const notes = Object.keys(this.faultNotes)
```

Comment:
Could use `Object.values` instead? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values

Reply:
Attempted to use this first off, but we don't have support, we'd need to configure TS for ES2017. Happy to make the change if that's what we want.

Reply:
ok nvm

Reply:
Upgraded to ES2017, confirmed that `Object.values` is working as expected.